dashboard/app/views/levels/editors/_md_instructions.haml

```diff
@@ -1,4 +1,7 @@
 .field
+  = f.label :instructions
+  = f.text_area :instructions, placeholder: 'Instructions', rows: 4
+.field
   = f.label :markdown_instructions
   %p When specified, these instructions will render in the instructions popup.
   %p The regular instruction field above will still display as the instructions below the game area.
```

Comment:
Yep, this is true for Widgets as well!  The markdown will show in the instructions dialog and the non-markdown instructions will show below the visualization (for pixelation) or above the widget (for text-compression).